README.md

```diff
@@ -15,8 +15,9 @@ Scapy is a powerful Python-based interactive packet manipulation program and
 library.
 
 It is able to forge or decode packets of a wide number of protocols, send them
-on the wire, capture them, match requests and replies, and much more. It
-designed to allow fast packet prototyping by using default values that work.
+on the wire, capture them, store or read them using pcap files, match requests
+and replies, and much more. It designed to allow fast packet prototyping by using
```

Comment:
The mistake was there before your edit, but still, I suppose "It **is** designed"